octobot_evaluators/channels/matrix.py

```diff
@@ -46,14 +46,16 @@ async def send(self,
                    exchange_name=None,
                    cryptocurrency=CHANNEL_WILDCARD,
                    symbol=CHANNEL_WILDCARD,
-                   time_frame=None):
+                   time_frame=None,
+                   origin_consumer=None):
```

Comment:
I think we should add this param to ```send_eval_note``` as well: it's the one that is making strategies call themselves

Reply:
right :+1: